torch/fx/passes/split_utils.py

```diff
@@ -5,8 +5,10 @@
 import torch.nn as nn
 from torch.fx.graph import map_arg
 from .tools_common import NodeList, NodeSet
+from torch.fx._compatibility import compatibility
 
 
+@compatibility(is_backward_compatible=False)
```

Comment:
cc @842974287 FYI I'm conservatively setting backward-compatibility for the APIs in the file to `False`. Please feel free to flip this to `True` if/when you feel comfortable providing stability guarantees for the API.